examples/md-flexible/tests/domainDecomposition/MixedBoundaryConditionTest.cpp

```diff
@@ -41,43 +41,53 @@ void initializeAutoPasContainer(RegularGridDecomposition::SharedAutoPasContainer
 }  // namespace
 
 /**
- * Simple test designed to show that reflection at a corner between 2 reflective faces works correctly
+ * Simple test designed to show that reflection at a corner between 2 reflective faces works correctly.
+ * Places particles within reflective skin and tests for correct reflective behaviour including for:
+ * - cases where the particle's velocity is away from the boundary
+ * - cases where the particle is in a corner, including
+ *   + reflective/reflective
+ *   + reflective/periodic
+ *   + reflective/reflective/periodic
+ * - cases for both upper and lower boundaries
  */
 TEST_F(MixedBoundaryConditionTest, testSimpleReflection) {
-  std::vector<std::string> arguments = {"md-flexible", "--yaml-filename",
-                                        std::string(YAMLDIRECTORY) + "reflectionTest.yaml"};
-
-  char *argv[3] = {arguments[0].data(), arguments[1].data(), arguments[2].data()};
-
-  MDFlexConfig configuration(3, argv);
-
-  std::array<double, 3> localBoxMin = configuration.boxMin.value;
-  std::array<double, 3> localBoxMax = configuration.boxMax.value;
-
-  RegularGridDecomposition domainDecomposition(configuration.boxMin.value, configuration.boxMax.value,
-                                               configuration.subdivideDimension.value, configuration.cutoff.value,
-                                               configuration.verletSkinRadius.value, configuration.reflWidth.value,
-                                               configuration.boundaryOption.value);
+  // initialise AutoPas container & domainDecomposition
+  const std::array<double,3> boxMin = {0.,0.,0.};
+  const std::array<double,3> boxMax = {5.,5.,5.};
+  const std::array<double,3> boxLength = {boxMax[0] - boxMin[0],boxMax[1] - boxMin[1],boxMax[2] - boxMin[2]};
```

Comment:
we actually have some array math implemented that you could use here:
```suggestion
  const std::array<double,3> boxLength = autopas::utils::ArrayMath::sub(boxMax, boxMin);
```
If you would use this `sub()` multiple times in a function or find the namespace chain annoying you could also place a `using autopas::utils::ArrayMath::sub;` e.g. at the top of this function. However, never do this on a class-wide or, even worse project-wide scope.